routers/web/admin/users.go

```diff
@@ -125,10 +125,14 @@ func NewUserPost(ctx *context.Context) {
 		Name:      form.UserName,
 		Email:     form.Email,
 		Passwd:    form.Password,
-		IsActive:  true,
 		LoginType: auth.Plain,
 	}
 
+	overwriteDefault := &user_model.CreateUserOverwriteOptions{
+		IsActive:   util.OptionalBoolTrue,
```

Comment:
OK to bypass setting.Service.RegisterEmailConfirm and setting.Service.RegisterManualConfirm here?

Reply:
OK as per https://github.com/go-gitea/gitea/pull/19310#discussion_r842302420